StratisCore.UI/src/app/wallet/smart-contracts/components/modals/transaction/transaction.component.html

```diff
@@ -17,30 +17,30 @@ <h5 class="modal-title">{{title}}</h5>
                 </div>
                 <div class="form-group clearfix" data-toggle="tooltip" data-placement="right" title="The amount of {{ coinUnit }} to send with this transaction">
                     <label for="amount">Amount</label>
-                    <input formControlName="amount" id="amount" type="number" step="0.01" class="form-control" placeholder="Amount" [class.is-invalid]="amount.invalid && (amount.dirty || amount.touched)">
+                    <input formControlName="amount" id="amount" type="number" step="0.01" class="form-control" placeholder="Amount" [class.is-invalid]="amount.invalid && (amount.dirty || amount.touched)" min="0">
```

Comment:
min=0 should actually be checked in transaction.component.ts using a validator like we do throughout the rest of the wallet.
This is beyond the scope of the current issue though, but needs changing throughout the entire smart contracts component.

Reply:
I will add a separate issue for this.